SwrveSDKCommon/src/main/java/com/swrve/sdk/SwrveHelper.java

```diff
@@ -201,4 +203,10 @@ public static boolean hasFileAccess(String filePath) {
             return false;
         }
     }
+
+    public static float convertDipToPixels(Context context, float dipValue) {
+        DisplayMetrics metrics = context.getResources().getDisplayMetrics();
+        float pixelValue = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dipValue, metrics);
```

Comment:
I like to return a variable value so I can breakpoint at the return and see exactly the value that is being returned. Otherwise if you breakpoint at a return expression, you have to evaluate it.